PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -318,12 +424,15 @@ LoadAndSetupImage (
   }
   AddMeasurePoint (0x4070);
 
-  if ((LoadedImage->Flags & LOADED_IMAGE_IAS) != 0) {
+  if (FeaturePcdGet (PcdPodsBootEnabled)) {
```

Comment:
can we do similar check as IAS like :
if ((LoadedImage->Flags & LOADED_IMAGE_PODS) != 0)   ?